src/ui/buttons/props.js

```diff
@@ -391,7 +392,8 @@ export function normalizeButtonStyle(props : ?ButtonPropsInputs, style : ButtonS
         shape = fundingConfig.shapes[0],
         tagline = (layout === BUTTON_LAYOUT.HORIZONTAL && !fundingSource),
         height,
-        period
+        period,
+        menuDirection = MENU_PLACEMENT.BELOW
```

Comment:
Should this be `menuPlacement`?
```suggestion
        menuPlacement = MENU_PLACEMENT.BELOW
```

Reply:
oh shoot thanks!